features/fixtures/ios/build.sh

```diff
@@ -5,8 +5,44 @@ echo "$(sw_vers -productName) $(sw_vers -productVersion) $(sw_vers -buildVersion
 
 set -euxo pipefail
 
+disable_swizzling_key='<key>test_key_to_replace1</key>'
+disable_swizzling_value='<string>test_value_to_replace1</string>'
+swizzling_premain_key='<key>test_key_to_replace2</key>'
+swizzling_premain_value='<string>test_value_to_replace2</string>'
+
+fixture_name='Fixture'
+for ((i=1;i<=$#;i++));
+do
+    if [ ${!i} = '--fixtureName' ]
+    then ((i++))
+        fixture_name=${!i};
+    fi
+    if [ ${!i} = '--disableSwizzling' ]
+    then
+        disable_swizzling_key='<key>disableSwizzling</key>'
+        disable_swizzling_value='<true/>\t\t\t\t\t\t\t\t\t\t';
+    fi
+    if [ ${!i} = '--swizzlingPremain' ]
+    then
+        swizzling_premain_key='<key>swizzleViewLoadPreMain</key>'
+        swizzling_premain_value='<true/>\t\t\t\t\t\t\t\t\t'
+    fi
+done;
+
+sed -i '' -e 's|DISABLE_SWIZZLING_KEY|'$disable_swizzling_key'|' $(dirname "${BASH_SOURCE[0]}")/Fixture/Info.plist
+sed -i '' -e 's|DISABLE_SWIZZLING_VALUE|'$disable_swizzling_value'|' $(dirname "${BASH_SOURCE[0]}")/Fixture/Info.plist
+sed -i '' -e 's|SWIZZLING_PREMAIN_KEY|'$swizzling_premain_key'|' $(dirname "${BASH_SOURCE[0]}")/Fixture/Info.plist
+sed -i '' -e 's|SWIZZLING_PREMAIN_VALUE|'$swizzling_premain_value'|' $(dirname "${BASH_SOURCE[0]}")/Fixture/Info.plist
```

Comment:
For the default build, you end up with these entries in the plist.  I guess it doesn't matter, but as the intention to simply remove  the placeholders?
<img width="303" alt="image" src="https://github.com/bugsnag/bugsnag-cocoa-performance/assets/5239394/ddca2da2-d952-4150-8cab-6a894aab639b">

Reply:
Yes, that was the intention. I removed the placeholders